appender.go

```diff
@@ -42,6 +42,45 @@ func TestAppender(t *testing.T, store types.Storager) {
 			})
 		})
 
+		Convey("When CreateAppend with an existing appendable object", func() {
+			path := uuid.NewString()
+			o, err := ap.CreateAppend(path)
+
+			defer func() {
+				err := store.Delete(path)
+				if err != nil {
+					t.Error(err)
+				}
+			}()
+
+			Convey("The first returning error should be nil", func() {
+				So(err, ShouldBeNil)
+			})
+
+			size := rand.Int63n(4 * 1024 * 1024) // Max file size is 4MB
+			r := io.LimitReader(randbytes.NewRand(), size)
+
+			_, err = ap.WriteAppend(o, r, size)
+			if err != nil {
+				t.Fatal(err)
+			}
```

Comment:
Do we need to `CommitAppend` now?

Reply:
Yes, you're right. We need `CommitAppend` here as for dropbox `path` will be assigned when `CommitAppend`. If we miss `CommitAppend` here, there will be no association between the two append operations and the first append object will not be deleted.